Gemfile.lock

```diff
@@ -1,46 +1,48 @@
 GIT
   remote: https://github.com/CocoaPods/CocoaPods.git
-  revision: 10b69dbd9b991442646944f118f569d855652b26
+  revision: 783424de392ebb327f7dc25d685b8df9605e76be
+  tag: 1.6.0.rc.2
   specs:
-    cocoapods (1.5.3)
+    cocoapods (1.6.0.rc.2)
```

Comment:
I believe this will break Auth on tvOS and maybe some other travis tests.  Hopefully I'll be able to sort those out before we're ready to merge this to master.